extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-slider.js

```diff
@@ -78,5 +78,47 @@ describes.realWin(
         ampStorySlider.getRootElement().querySelector('input[type="range"]')
       ).to.not.be.null;
     });
+
+    it('should be disabled after the input event', async () => {
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const slider = ampStorySlider
+        .getRootElement()
+        .querySelector('input[type="range"]');
+      // simulates a change event, which is when the user releases the slider
+      slider.dispatchEvent(new CustomEvent('change'));
+      expect(slider.hasAttribute('disabled')).to.be.true;
+    });
+
+    it('checks that input is the same as the number displayed in the bubble', async () => {
```

Comment:
Tests should be called "should ... when ...". Eg: it("should display the percentage in the bubble when the user drags the slider") / "should set the --percentage variable when the user drags"